packages/docs/docs/app/app-introduction/index.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 0
+tags:
+  - app
+---
+
+# Introduction
+
+The Medplum App is a user-facing web application available at <https://app.medplum.com/> and is targeted towards developers and project administrators. This administrative app is developed from the [Medplum React Components](/docs/ui-components), meaning that many of these views and functionality can be embedded into your own custom apps.
+
+This guide will go over how to navigate the basic functionality of the Medplum App.
+
+## Homepage
+
+![App Homepage](./app-homepage.png)
+
+When you first log in to the Medplum App, you'll see the page shown above. This home page consists of a few primary components:
+
+- **[Search Control](#search-control)** - Displays a list of the currently selected resource type.
+- **[Sidebar](#sidebar)** - Used to select the current resource type and quickly navigate to different parts of the Medplum App.
+- **[Quick Search Bar](#quick-search-bar)** - Used to search for specific resource instances.
+- **[Profile Selector](#profile-selector)** - Displays profile information for the currently logged in user.
+
+## Search Control
+
+The Search Control displays a paginated list of all the resources for the selected resource type. You can select individual resources for certain operations using the checkbox on the left hand, and you can select all resources on the current page by clicking on the checkbox in the table header.
+
+The toolbar at the top contains actions that you can use to customize this view.
```

Comment:
Consider adding the "toolbar" to graphic above as on the first read it is tricky to piece together what this is